MetaMask/metamask-extension
# E2e json rpc method tests

## **Description**
_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:_
_1. What is the reason for the change?_
_2. What is the improvement/solution?_

Adds e2e tests for the json rpc methods: `eth_coinbase`, `eth_requestAccounts`, `wallet_requestPermissions`, and `eth_subscribe`.

Also use the defaultGanache options in other json rpc tests when appropriate.

## **Manual testing steps**

_1. Pull down this branch
_2. `yarn && yarn build:test && yarn test:e2e:chrome:rpc`
_3. . Tests suite should not fail.

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**

_[screenshot]_

### **After**

_[screenshot]_

## **Related issues**

_Fixes #???_

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
